packages/notification-services-controller/tsconfig.build.json

```diff
@@ -17,5 +17,11 @@
       "path": "../keyring-controller/tsconfig.build.json"
     }
   ],
-  "include": ["../../types", "./src"]
+  "include": ["../../types", "./src"],
+  "exclude": [
+    "./jest.config.packages.ts",
+    "**/*.test.ts",
+    "**/jest.config.ts",
+    "**/__fixtures__/"
+  ]
```

Comment:
Only really wanted to include the `**/__fixtures__/` folder, but it overwrites the exclude folder that we extend from `tsconfig.packages.build.json`
https://github.com/MetaMask/core/blob/20a17426d732d41e438c74b7a3193ea6e193e3d7/tsconfig.packages.build.json#L10